triage/views.py

```diff
@@ -95,7 +96,46 @@ def dispatch(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        persona = forms.get_persona(self.triage_answers)
-        context['persona'] = persona
         context['triage_result'] = self.triage_answers
+        context['section_configuration'] = self.get_section_configuration()
         return context
+
+    def get_section_configuration(self):
+        persona = forms.get_persona(self.triage_answers)
+        if persona == forms.NEW_EXPORTER:
+            return self.get_persona_new_section_configuration()
+        elif persona == forms.OCCASIONAL_EXPORTER:
+            return self.get_persona_occasional_section_configuration()
+        elif persona == forms.REGULAR_EXPORTER:
+            return self.get_persona_regular_section_configuration()
+
+    def get_persona_new_section_configuration(self):
+        return {
+            'persona_article_group': structure.PERSONA_NEW_ARTICLES,
+            'trade_profile': not forms.get_is_sole_trader(self.triage_answers),
```

Comment:
IMHO methods without args/kwargs that are later accessing `self` are bit implicit, but I see the point here is to keep `get_section_configuration` cleaner

Reply:
:+1:  I'll pass in self.triage_answers as a kwarg and turn these into static functions.